Vagrantfile

```diff
@@ -163,6 +167,23 @@ def suse_common(config)
     java_package: "java-1_8_0-openjdk-devel")
 end
 
+def sles_common(config)
```

Comment:
Much of this is the same as suse_common - could suse common grow and extra parameter that handles this?